src/lib/header-bidding/a9/a9.ts

```diff
@@ -70,11 +64,23 @@ const requestBids = async (
 		return requestQueue;
 	}
 
+	const section = window.guardian.config.page.section;
+
+	const isNetworkFront =
+		section === 'uk' || section === 'us' || section === 'au';
```

Comment:
I am trying to check in the Trello ticket if those info are there but just double checking you got them from Alex? I mean are those the only three regions that we are showing a9 in Fronts?

Reply:
missing europe here

Reply:
@deedeeh thats right `inline-1` slots only render on mobile, assuming it was an omission, and double checked with @Amouzle.

Reply:
> missing europe here

@Jakeii assuming `international` as well as `europe`?